src/Npgsql/Schema/NpgsqlDbColumn.cs

```diff
@@ -50,7 +50,7 @@ public NpgsqlDbColumn()
         /// <inheritdoc cref="DbColumn.BaseCatalogName" />
         public new string BaseCatalogName
         {
-            get => base.BaseCatalogName;
+            get => base.BaseCatalogName!;
```

Comment:
Why a bang instead of adding the nullability attribute to the return type?

Reply:
The idea is that with Npgsql specifically, the BaseCatalogName (which contains the database name) is never supposed to be null. So if someone is coding against NpgsqlDbColumn (as opposed to DbColumn) they're never supposed to see null on this property.

Reply:
So why not just do
```csharp
[NotNull] public override string? BaseCatalogName { get => ... }
```
🤣

Reply:
Ah, I see, it's not virtual.